.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js: 
   - 10.8
+before_install:
+  - npm i -g npm@6.9.0  
```

Comment:
Hmm, I don't know what we want to have to do this ourselves. Why is this required?

Reply:
A couple of the updated dependencies have aliases which were introduced in npm version 6.9.0 (which is a couple of years old). The version of Travis we are running uses outdated versions of node and npm (current recommended node is 12.18.3 with npm 6.14.6). We may be able to just change the version of node and remove that before_install clause. Ultimately what we should do depends on if we are using an older version of npm and node for a specific reason.

Reply:
Circling back here, can we just change the node version above?